pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1073,6 +1078,15 @@ func getKafkaConfig(kafka *api.KafkaExtObject) {
 	}
 }
 
+func getKafkaJBPMConfig(kafkaJbpmEmitter *api.KafkaJBPMEventEmittersObject) {
+	if kafkaJbpmEmitter != nil {
+		//if something is missing we set mandatory defaults
+		if kafkaJbpmEmitter.MaxBlockMs == nil {
+			kafkaJbpmEmitter.MaxBlockMs = Pint32(2000)
+		}
```

Comment:
We don't need to set this default by ourself, this is already part of KafkaEventEmitter.java. I think it's enough to have it mentioned just in parameter description in KieApp.yaml file.

Reply:
Ok